scripts/main/dragDrop.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
A general comment:

The front end tends to avoid global function names like this. Maybe have it as `dragdrop.start` and `dragdrop.finish` or something like that? Although, given how short this file is, I'm hesitating if it wouldn't make more sense to simply add these two functions to an existing file, such as `lychee.js`?